distros/kubernetes/nvsentinel/charts/health-events-analyzer/values.yaml

```diff
@@ -32,15 +32,156 @@ resources:
 logLevel: info
 
 config: |
-  [[rules]]
-  # The node condition for this rule needs to be removed manually because health-events-analyzer does not publish healthy events to clear it.
+  # The node condition for these rules needs to be removed manually because health-events-analyzer does not publish healthy events to clear it.
   # Please run the command below to remove the node condition:
-  # kubectl get node <NODE_NAME> -o json | jq '.status.conditions |= map(select(.type != "MultipleRemediations"))' | kubectl replace -f - --subresource=status
+  # kubectl get node <NODE_NAME> -o json | jq '.status.conditions |= map(select(.type != "<NAME_OF_APPLIED_RULE>"))' | kubectl replace -f - --subresource=status
+  [[rules]]
   name = "MultipleRemediations"
   description = "Detect if multiple remediations are performed within 7 days on a node"
-  time_window = "168h"
   recommended_action = "CONTACT_SUPPORT"
+  stage = [
+    '''
+    {
+      "$match": {
+        "$expr": {
+          "$gte": [
+            "$healthevent.generatedtimestamp.seconds",
+            {"$subtract": [{"$divide": [{"$toLong": "$$NOW"}, 1000]}, 604800]}
+          ]
+        }
+      }
+    }
+    ''',
+    '''
+    {
+      "$match": {
+        "healtheventstatus.faultremediated": true,
+        "healthevent.nodename": "this.healthevent.nodename",
+        "healthevent.isfatal": "this.healthevent.isfatal"
+      }
+    }
+    ''',
+    '{"$count": "count"}',
+    '{"$match": {"count": {"$gte": 5}}}'
+  ]
 
-  [[rules.sequence]]
-  criteria = {"healtheventstatus.faultremediated" = true, "healthevent.nodename" = "this.healthevent.nodename", "healthevent.isfatal" = "this.healthevent.isfatal"}
-  errorCount = 5
+  [[rules]]
+  name = "RepeatedXidError"
+  description = "Detect occurrence of fatal XIDs 5 times within 24 hours"
+  recommended_action = "CONTACT_SUPPORT"
+  stage = [
+    '''
+    {
+      "$match": {
+        "$expr": {
+          "$gte": [
+            "$healthevent.generatedtimestamp.seconds",
+            {"$subtract": [{"$divide": [{"$toLong": "$$NOW"}, 1000]}, 86400]}
+          ]
+        }
+      }
+    }
+    ''',
+    '''
+    {
+      "$match": {
+        "healthevent.ishealthy": false,
+        "healthevent.entitiesimpacted.0.entitytype": "GPU",
+        "healthevent.entitiesimpacted.0.entityvalue": "this.healthevent.entitiesimpacted.0.entityvalue",
```

Comment:
This seems to assume that entity 0 will always be GPU, can we change that? What if entity 0 is NVLink and entity 1 is GPU?